ansible/group_vars/all.yml

```diff
@@ -47,15 +47,26 @@ local_storage_nodes:
   - storagenode-3
 local_storage_node_label: 'localstorage: "true"'
 local_storage_instances:
-  key: localstorage
-  value: "true"
-  volumes:
-  - /dev/vg_ls/lv_tv00
-  - /dev/vg_ls/lv_tv01
-  - /dev/vg_ls/lv_tv02
-  - /dev/vg_ls/lv_tv03
-  - /dev/vg_ls/lv_tv04
-  - /dev/vg_ls/lv_tv05
+  - key: localstorage
```

Comment:
I think the message got lost when I provided feedback last time but we will have to modify the `localvolume.yml.j2` [template](https://github.com/redhat-performance/webfuse/blob/main/ansible/roles/local_storage/templates/localvolume.yml.j2) and [tasks](https://github.com/redhat-performance/webfuse/blob/main/ansible/roles/local_storage/tasks/main.yml#L16) to support this new var layout for deploying localvolumes.

The task should iterate over the list and apply the template each time.  The task should have a with_items statement something like: `with_items: "{{local_storage_instances}}" ` and then the template will need to be modified to reference each `item.variable`

Example:
```
---
apiVersion: "local.storage.openshift.io/v1"
kind: "LocalVolume"
metadata:
  name: "ld-{{item.storageclassname}}"
  namespace: "openshift-local-storage" 
spec:
  nodeSelector: 
    nodeSelectorTerms:
    - matchExpressions:
      - key: "{{item.key}}"
...
```

And so forth. Let me know if that helps or not.

Reply:
I took a different approach, have a look at the template and let me know what you think. I couldn't test it yet, so it's possible that I missed something, but I think it should work.